### DISCUSSION TITLE
Generic base connection type in Airflow

### DISCUSSION THREAD

**Discussion by match-gabeflores** [apache/airflow#20350] [Ideas] [answered]
we have a secret key/password that we'd like to store in Airflow for connections (and we would retrieve this info via BaseHook.get_connection). I was hoping to see a Generic connection type there but don't see one. 

Does this sound like something that could be added. 

Any workarounds? Otherwise, we'll do something like an environment variable.

**Comment by potiuk** [comment]
Use any connection - HTTP?

**Comment by potiuk** [comment]
And if you want to add a genric one - feel free to add PR to add it: https://github.com/apache/airflow/blob/17404f1f10efd41f98eb8a0317b578ff40f9c77d/airflow/www/views.py#L3374

**Comment by malthe** [comment, marked as answer]
@match-gabeflores I have suggested this in https://github.com/apache/airflow/pull/22310 now.

**Reply by match-gabeflores** [reply]
Thank you!
